core/src/main/scala/cats/std/bigInt.scala

```diff
@@ -1,7 +1,14 @@
 package cats
 package std
 
-trait BigIntInstances extends algebra.std.BigIntInstances {
+trait BigIntInstances {
+  implicit val bigIntAlgebra: BigIntAlgebra =
```

Comment:
I think this is out of the scope of this PR, but just to jot it down while I'm thinking of it: is there any benefit to having this statically typed to `BitIntAlgebra` instead of just `Order[BigInt]`?